lib/cartopy/tests/crs/test_oblique_mercator.py

```diff
@@ -0,0 +1,193 @@
+            (754893.24196, 3626768.52447),
+            (1035451.51708, 3595849.69689),
+        ),
+        ParamTuple(
+            "scale_factor",
+            # Number inherited from test_mercator.py .
+            dict(scale_factor=0.939692620786),
+            dict(k="0.939692620786"),
+            (-230325.01183, 5287432.86131),
+            (33313.52899, 5258378.6672),
+        ),
+        ParamTuple(
+            "globe",
+            dict(globe=ccrs.Globe(ellipse="sphere")),
+            dict(ellps="sphere"),
+            (-244502.86059, 5646357.44304),
+            (35364.23322, 5615460.21872),
+        ),
+        ParamTuple(
+            "combo",
+            dict(
+                azimuth=90.0,
+                central_longitude=90.0,
+                central_latitude=45.0,
+                false_easting=1000000,
+                false_northing=-2000000,
+                scale_factor=0.939692620786,
+                globe=ccrs.Globe(ellipse="sphere"),
+            ),
+            dict(
+                alpha="89.999",
+                lonc="90.0",
+                lat_0="45.0",
+                x_0="1000000",
+                y_0="-2000000",
+                k="0.939692620786",
+                ellps="sphere",
+            ),
+            (-4279982.08123, 1916861.68937),
+            (-4138080.80706, 1631302.04295),
+        ),
+    ]
+    param_ids: List[str] = [p.id for p in param_list]
+
+    @pytest.fixture(autouse=True, params=param_list, ids=param_ids)
+    def make_variant_inputs(self, request) -> None:
+        inputs: TestCrsArgs.ParamTuple = request.param
+
+        self.oblique_mercator = ccrs.ObliqueMercator(**inputs.crs_kwargs)
+        proj_kwargs_expected = dict(
+            ellps="WGS84",
+            lonc="0.0",
+            lat_0="0.0",
+            k="1.0",
+            x_0="0.0",
+            y_0="0.0",
+            alpha="0.0",
+            units="m",
+        )
+        proj_kwargs_expected.update(inputs.proj_kwargs)
+        self.proj_params = {
+            f"{k}={v}" for k, v in proj_kwargs_expected.items()
+        }
+
+        self.expected_a = inputs.expected_a
+        self.expected_b = inputs.expected_b
+
+    def test_proj_params(self):
+        check_proj_params("omerc", self.oblique_mercator, self.proj_params)
+
+    def test_transform_point(self, plate_carree):
+        # (Point equivalence has been confirmed via plotting).
+        src_expected = (
+            (self.point_a_plate_carree, self.expected_a),
+            (self.point_b_plate_carree, self.expected_b),
+        )
+        for src, expected in src_expected:
+            res = self.oblique_mercator.transform_point(
+                *src,
+                src_crs=plate_carree,
+            )
+            np.testing.assert_array_almost_equal(res, expected, decimal=4)
+
+
+@pytest.fixture
+def oblique_variants(
+    oblique_mercator,
+    rotated_mercator,
+) -> Tuple[ccrs.ObliqueMercator, ccrs.ObliqueMercator, ccrs.ObliqueMercator]:
```

Comment:
@trexfeathers Minor point, but didn't know if you wanted to create a [typing alias](https://docs.python.org/3/library/typing.html#type-aliases) for this?

Admittedly, this is a bit of a marmite comment. So happy for you to say "nope" :+1:

Reply:
Well it's certainly good to know that's an option. I know what I've written is ugly, but I think an alias in this case would be another type of ugly! So, "nope" 😄